packages/common/src/scss/_typography.scss

```diff
@@ -1,6 +1,4 @@
 @use 'sass:math';
-@tailwind base;
-@tailwind utilities;
```

Comment:
Are these just not needed? I don't see them re-added anywhere else.

Reply:
They were redundant!